src/routes/events/containers/EventsPage.js

```diff
@@ -38,9 +38,7 @@ export class EventsPage extends React.Component<PropsType> {
     <EventListItem event={event} language={language} key={event.path} />
 
   redirectToPath = (path: string) => {
-    const action = pathToAction(path, this.props.routesMap)
-    setKind(action, 'push')
-    this.props.dispatch(action)
+    this.props.redirect(pathToAction(path, this.props.routesMap))
```

Comment:
this does not push the url on top of the stack, but replace the current entry with `path`.
(So if you click on a link and navigate back, the browser will navigate two pages back)